game-core/src/main/java/games/strategy/triplea/TripleAPlayer.java

```diff
@@ -711,10 +711,10 @@ public boolean confirmMoveKamikaze() {
   }
 
   @Override
-  public boolean confirmMoveHariKari() {
+  public void confirmMoveHariKari() {
     final String question =
         "All units in destination territory will automatically die, do you still want to move?";
-    return ui.getOk(question);
+    ui.getOk(question);
```

Comment:
This seems odd.
This smells like the original plan of the code designer was to query this method to get the confirmation, but it looks like the UI handles the result automatically